cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -313,14 +321,129 @@ func (oi *OVFImporter) importDiskFilesToImages() error {
+			return err
+		}
+	}
+
+	// Import boot disk image with attached disks
+	if len(diskInfos) > 0 {
+		if err = oi.importBootDiskImage(diskInfos[0], disksNamesPrefix); err != nil {
+			return err
+		}
+	}
+
+	return err
+}
+
+func (oi *OVFImporter) importBootDiskImage(bootDiskInfos ovfutils.DiskInfo, disksNamesPrefix string) error {
+	// Import boot disk image with attached disks
+
+	oi.Logger.User(fmt.Sprint("Importing boot Disk Image ..."))
+
+	// create attached disks objects from the created data disks
+	var attachedDisks []*compute.AttachedDisk
+	for _, disk := range oi.disks {
+		attachedDisks = append(attachedDisks, &compute.AttachedDisk{Source: disk.GetURI()})
+	}
+
+	bootDiskFilePath := bootDiskInfos.FilePath
+	bootDiskFilePathSource, err := importer.NewSourceFactory(oi.storageClient).Init(bootDiskFilePath, "")
+	if err != nil {
+		return err
+	}
+
+	imageName := fmt.Sprintf("%s-%s", disksNamesPrefix, "boot-image")
+	request := importer.ImageImportRequest{
+		ExecutionID:                   imageName,
+		CloudLogsDisabled:             oi.params.CloudLogsDisabled,
+		ComputeEndpoint:               oi.params.Ce,
+		ComputeServiceAccount:         oi.params.ComputeServiceAccount,
+		WorkflowDir:                   oi.params.WorkflowDir,
+		GcsLogsDisabled:               oi.params.GcsLogsDisabled,
+		ImageName:                     imageName,
+		Network:                       oi.params.Network,
+		NoExternalIP:                  oi.params.NoExternalIP,
+		NoGuestEnvironment:            oi.params.NoGuestEnvironment,
+		Oauth:                         oi.params.Oauth,
+		Project:                       *oi.params.Project,
+		ScratchBucketGcsPath:          pathutils.JoinURL(oi.params.ScratchBucketGcsPath, imageName),
+		Source:                        bootDiskFilePathSource,
+		StdoutLogsDisabled:            oi.params.StdoutLogsDisabled,
+		Subnet:                        oi.params.Subnet,
+		Timeout:                       oi.params.Deadline.Sub(time.Now()),
+		Tool:                          oi.params.GetTool(),
+		UefiCompatible:                oi.params.UefiCompatible,
+		Zone:                          oi.params.Zone,
+		TanslationWorkerAttachedDisks: attachedDisks,
+	}
+	request.OS = oi.params.OsID
+	request.BYOL = oi.params.BYOL
+	importer.FixBYOLAndOSArguments(&request.OS, &request.BYOL)
+
+	if oi.imageImporter == nil {
+		oi.imageImporter, err = importer.NewImporter(request, oi.computeClient, oi.storageClient, oi.Logger)
+		if err != nil {
+			return err
+		}
+	}
+
+	err = oi.imageImporter.Run(oi.ctx)
+	if err != nil {
+		return err
+	}
+
+	oi.images = []domain.Image{image.NewImage(request.Project, request.ImageName)}
+
+	return nil
+}
+
+func (oi *OVFImporter) createDisksFromVmdkFiles(diskInfos []ovfutils.DiskInfo, dataDiskPrefix string) error {
+	// create disk for each vmdk file
+
+	params := *oi.params
+
+	var disks []domain.Disk
+	for i, info := range diskInfos {
```

Comment:
It looks like you're creating blank disks here; don't you want to write the OVF file's VMDK contents to the disk?